Factorio/fctrserver

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+# Factorio
+# Server Management Script
+# Author: Daniel Gibbs
+# Contributor: Kristian Polso
+# Website: https://gameservermanagers.com
+if [ -f ".dev-debug" ]; then
+        exec 5>dev-debug.log
+        BASH_XTRACEFD="5"
+        set -x
+fi
+
+version="171016"
+
+#### Variables ####
+
+# Notification Alerts
+# (on|off)
+
+# Email
+emailalert="off"
+email="email@example.com"
+#emailfrom="email@example.com"
+
+# Pushbullet
+# https://www.pushbullet.com/#settings
+pushbulletalert="off"
+pushbullettoken="accesstoken"
+
+# Server settings
+# More settings available after install in serverfiles/data/server-settings.json
+updateonstart="off"
+port="34197"
+rconport="34198"
+rconpassword="factorio"
+
+fn_parms(){
+parms="--start-server ${filesdir}/save1.zip --server-settings ${servercfgfullpath} --port ${port} --rcon-port ${rconport} --rcon-password ${rconpassword}"
```

Comment:
`--start-server <save>` could be replaced with `--start-server-load-latest` to make things easier.
Also means you can have autosaves enabled and rotating and the next start will automatically load the most recent.